### DISCUSSION TITLE
VR Payment / OPPWA Payment Plugin

### DISCUSSION THREAD

**Discussion by fabm3n** [pretix/pretix#3057] [Q&A] [answered]
Dear all,

I'm searching the VR Payment Plugin, but there are no Instructions on the Marketplace (https://marketplace.pretix.eu/products/oppwa/) and I couldn't find on PyPI

Is it an Enterprise only Plugin?

Best regards
fabm3n

**Comment by raphaelm** [comment]
Hmm, I think we just never got around to make the decision on whether to publish that plugin. I think we could, @pc-coholic any reasons we shouldn't?

**Reply by pc-coholic** [reply]
I don't mind - release away :-)

**Reply by fabm3n** [reply]
Thanks, that would be great!

**Comment by raphaelm** [comment, marked as answer]
Released now!

https://github.com/pretix/pretix-oppwa
https://pypi.org/project/pretix-oppwa/
